packages/compass/webpack.config.js

```diff
@@ -38,10 +42,16 @@ module.exports = (_env, args) => {
     // Runtime implementation depends on worker file existing near the library
     // main import and for that reason it needs to stay external to compass (and
     // compass-shell plugin)
-    '@mongosh/node-runtime-worker-thread':
+    '@mongosh/node-x-worker-thread':
```

Comment:
Why do you need to change this to `x`?

Reply:
definitely unintended, i should have opened this in WIP for 5 mins, i'm still going trough the changes myself